demo/admin/eslint.config.mjs

```diff
@@ -0,0 +1,16 @@
+import cometConfig from "@comet/eslint-config/react.js";
```

Comment:
Comet config is kinda a reserved term for [comet-config.json](https://github.com/vivid-planet/comet/blob/main/demo/api/src/comet-config.json). Could we change this to something else? Maybe "preset"?

Reply:
what's your suggestion? 

`eslintCometConfig`?

Reply:
refactored naming, lets have a relook @johnnyomair

Reply:
yes, better!